pkg/es/config/config.go

```diff
@@ -412,7 +424,7 @@ func GetHTTPRoundTripper(c *Configuration, logger *zap.Logger) (http.RoundTrippe
 	return transport, nil
 }
 
-func loadToken(path string) (string, error) {
+func LoadFileContent(path string) (string, error) {
```

Comment:
TODO this should probably remain being called LoadToken, since it's not a plain file read